server.js

```diff
@@ -1,10 +1,27 @@
+const routes = require('./routes');
 const jsonServer = require('json-server');
 const server = jsonServer.create();
 const router = jsonServer.router('db.json');
 const middlewares = jsonServer.defaults();
 const port = process.env.PORT || 4000;
 
 server.use(middlewares);
+server.use(jsonServer.rewriter(routes));
+server.use(jsonServer.bodyParser);
+server.use((req, res, next) => {
+    // validate if request method is POST, PUT or DELETE and send direct status 200 success
+    // if catch not it, this clean the mock data with "{}" over property request
+    if (req.method === 'POST' || req.method === 'PUT' || req.method === 'DELETE') {
```

Comment:
add the patch request method

Reply:
**PATCH** request doesn't affect the mock data, that's why I didn't include it

Reply:
just add it because we will simulate the requests :P

Reply:
done